agent/stats/queue.go

```diff
@@ -343,6 +346,11 @@ func (queue *Queue) getCWStatsSet(f getUsageFloatFunc) (*ecstcs.CWStatsSet, erro
 		sum += thisStat
 	}
 
+	// don't emit metrics when sampleCount == 0
```

Comment:
what will happen on the TACS side is we send  metrics with sampleCount is 0?

Reply:
they will error, disconnect and send an invalid frame message to us